libbeat/tests/system/config/libbeat.yml.j2

```diff
@@ -0,0 +1,93 @@
+#================================ General =====================================
+
+# The name of the shipper that publishes the network data. It can be used to group
+# all the transactions sent by a single shipper in the web interface.
+# If this options is not defined, the hostname is used.
+name: {{shipper_name}}
+
+# The tags of the shipper are included in their own field with each
+# transaction published. Tags make it easy to group servers by different
+# logical properties.
+tags: [
+  {%- if agent_tags -%}
+      {%- for tag in agent_tags -%}
+          "{{ tag }}"
+          {%- if not loop.last %}, {% endif -%}
+      {%- endfor -%}
+  {%- endif -%}
+]
+
+{% if setup_template_name %}
+setup.template.name: setup_template_name
```

Comment:
Should these be a reference to the `setup_template_name ` variable?

Reply:
I would say yes. It got copied from here: https://github.com/elastic/beats/pull/5938/files#diff-bfed2fbdbe1d791a97c6ca3a63e6a8deL152

Will fix it for both variables.